### PR TITLE
Update to use new interface and amend readme and package.

### PR DESCRIPTION
**Pull request by e-e-e**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,47 +1,46 @@
...
+      })
+    },
+    write: function (req) {
+      if (verbose) console.log('trying to write', key, req.offset, req.data)
+      req.callback()
+    },
+    del: function (req) {
+      if (verbose) console.log('trying to del', key, req.offset, req.size)
+      req.callback()
     }
```
Comment (mafintosh) [inline index.js]: Don't implement write/del, then .deletable and .writable will be false :)

File: package.json [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
-    "url": "git+https://github.com/e-e-e/random-access-s3.git"
+    "url": "git+https://github.com/random-access-storage/random-access-s3.git"
   },
   "dependencies": {
-    "abstract-random-access": "^1.1.2",
-    "inherits": "^2.0.3"
+    "random-access-storage": "^1.1.0"
   },
   "devDependencies": {
+    "aws-sdk": "^2.188.0",
```
Comment (mafintosh) [inline package.json]: this is a prod dep right?

**Comment by mafintosh** [comment]
Nice! You might want to implement destroy as well (just delete the entire object on s3)

File: package.json (hunk 2 above)
Comment (e-e-e) [reply package.json]: ah yes it is! thank you for catching that one

**Comment by scriptjs** [comment]
Its looking good with the changes @mafintosh has recommended. Adding tests and travis will increase confidence for the package.

**Pull request merged**